### PR TITLE
Code for backing up incoming issues to a local db

### PR DESCRIPTION
**Pull request by hallvors**
Depends on #757 BTW.
Not tested :-o

### CODE REVIEW

**Comment by hallvors** [comment]
(Builds fail due to the dependency on the previous pull request)

**Comment by karlcow** [comment]
@hallvors to read https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md#submission-guidelines ;)

**Comment by hallvors** [comment]
So this code is a partial fix for https://github.com/webcompat/webcompat.com/issues/165#issuecomment-144733518 Todo list item 3 
I'll complete it and test it when #757 is merged because I'm not sure if it's a good idea to base one PR on a different PR. It becomes unclear what changes are reviewed where.

And @karlcow - "RTFM" isn't necessarily constructive feedback. I create PRs to get advice, code review, discussion and help. I've forgotten to mention #165  in the commit message, but apart from that I don't see what in the submission manual I should read again. Lack of local testing? Well, I might want advice on the direction we're taking even before I can actually run the code locally (due to aforementioned problem with depending on other pull requests and wanting to keep the changes and commits distinct).

**Comment by karlcow** [comment]
@hallvors to discuss the process in a teleconf this is out of scope for this orphan PR.

File: webcompat/helpers.py [hunk 1]
```diff
@@ -308,3 +308,20 @@ def get_comment_data(request_data):
...
+
+    URL in webcompat.com bugs follow this pattern:
+    **URL**: https://example.com/foobar
+    '''
+    url_pattern = re.compile('\*\*URL\*\*\: (.*)\n')
+    url_match = re.search(url_pattern, issue_body)
+    if url_match:
+        url = url_match.group(1).strip()
+        if not url.startswith(('http://', 'https://')):
+            url = "http://%s" % url
```
Comment (miketaylr) [inline webcompat/helpers.py]: Nit: I prefer the newer string format style: `'http://{0}'.format(url)`. But maybe that's just me.

File: webcompat/webhooks/helpers.py [hunk 2]
```diff
@@ -9,7 +9,11 @@
 import re
 import requests
 
+from db import issue_db
```
Comment (miketaylr) [inline webcompat/webhooks/helpers.py]: This doesn't exist yet, right?

Comment (hallvors) [inline webcompat/webhooks/helpers.py]: Correct. I'm going to add it now :)

Comment (miketaylr) [inline webcompat/webhooks/helpers.py]: Gotcha. I'm going to close this PR so someone doesn't accidentally merge this.

Feel free to send another one once it's ready to be merged (and feel free to ping for feedback on commit comments or in #165). :heart:

**Pull request closed without merging**